x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -25,6 +25,7 @@ export const LATEST_FINDINGS_INDEX_DEFAULT_NS =
   'logs-cloud_security_posture.findings_latest-default';
 
 export const BENCHMARK_SCORE_INDEX_TEMPLATE_NAME = 'logs-cloud_security_posture.scores';
+export const BENCHMARK_SCORE_INDEX_ROLLOVER_ALIAS = 'logs-cloud_security_posture.scores';
```

Comment:
`BENCHMARK_SCORE_INDEX_ROLLOVER_ALIAS` is the same as `BENCHMARK_SCORE_INDEX_TEMPLATE_NAME`, is this intentional?

Reply:
yep, I prefer to use dedicated constant for this purpose

Reply:
IMO a comment above the `rollover` field would do but if you keep this please add a comment explaining why we have 2 identical constants next to each other.

Reply:
done